sdk/agrifood/Azure.Verticals.AgriFood.Farming/tests/Generated/Samples/Samples_ApplicationData.cs

```diff
@@ -455,236 +435,308 @@ public async Task Example_Delete_AllParameters_Async()
+                Console.WriteLine(result[0].GetProperty("area").GetProperty("value").ToString());
+                Console.WriteLine(result[0].GetProperty("operationModifiedDateTime").ToString());
+                Console.WriteLine(result[0].GetProperty("operationStartDateTime").ToString());
+                Console.WriteLine(result[0].GetProperty("operationEndDateTime").ToString());
+                Console.WriteLine(result[0].GetProperty("attachmentsLink").ToString());
+                Console.WriteLine(result[0].GetProperty("associatedBoundaryId").ToString());
+                Console.WriteLine(result[0].GetProperty("partyId").ToString());
+                Console.WriteLine(result[0].GetProperty("id").ToString());
+                Console.WriteLine(result[0].GetProperty("eTag").ToString());
+                Console.WriteLine(result[0].GetProperty("status").ToString());
+                Console.WriteLine(result[0].GetProperty("createdDateTime").ToString());
+                Console.WriteLine(result[0].GetProperty("modifiedDateTime").ToString());
+                Console.WriteLine(result[0].GetProperty("source").ToString());
+                Console.WriteLine(result[0].GetProperty("name").ToString());
+                Console.WriteLine(result[0].GetProperty("description").ToString());
+                Console.WriteLine(result[0].GetProperty("createdBy").ToString());
+                Console.WriteLine(result[0].GetProperty("modifiedBy").ToString());
+                Console.WriteLine(result[0].GetProperty("properties").GetProperty("<key>").ToString());
             }
         }
 
         [Test]
         [Ignore("Only validating compilation of examples")]
         public async Task Example_GetAllApplicationData_Async()
         {
-            var credential = new DefaultAzureCredential();
-            var client = new FarmBeatsClient(credential).GetApplicationDataClient("2022-11-01-preview");
+            TokenCredential credential = new DefaultAzureCredential();
+            ApplicationData client = new FarmBeatsClient(credential).GetApplicationDataClient(apiVersion: "2022-11-01-preview");
 
-            await foreach (var item in client.GetAllApplicationDataAsync(3.14, 3.14, 3.14, 3.14, new string[] { "<sources>" }, new string[] { "<associatedBoundaryIds>" }, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, 3.14, 3.14, new string[] { "<applicationIds>" }, new string[] { "<names>" }, new string[] { "<propertyFilters>" }, new string[] { "<statuses>" }, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, DateTimeOffset.UtcNow, 1234, "<skipToken>", new RequestContext()))
+            await foreach (BinaryData item in client.GetAllApplicationDataAsync(null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null))
```

Comment:
well I have to say this is expected, for some reasons:
1. this is a sample that only takes minimum parameters.
2. these parameters are all optional (see code [here](https://github.com/Azure/azure-sdk-for-net/blob/ec26a4e9181413d90cb4becf54d63390b7363ecd/sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Generated/ApplicationData.cs#L394), there is no `Argument.AssertNotNull`)
3. they are converted to be required on the signature because we are avoiding possible ambiguous calls with the future convenience method.
Because of all these 3 reasons, we have to pass many `null` values here